### PR TITLE
docs: add K600 TV, new M350, MX Keys to devices.md and improve wording in it

### LINKED ISSUE #753: Logitech K600 is supported but not listed in devices.md
**Information**

The logitech K600 keyboard with touchpad is supported by solaar, but it is not listed in devices.md.
Here is the output for 'solaar show':

```
Unifying Receiver
  Device path  : /dev/hidraw0
  USB id       : 046d:c52b
  Serial       : E16D9B84
    Firmware   : 24.09.B0033
    Bootloader : 02.09
    Other      : AA.E5
  Has 1 paired device(s) out of a maximum of 6.
  Notifications: wireless (0x000100)
  Device activity counters: (empty)

  1: K600 TV
     Codename     : K600 TV
     Kind         : keyboard
     Wireless PID : 4078
     Protocol     : HID++ 4.5
     Polling rate : 8 ms (125Hz)
     Serial number: 202A5413
        Bootloader: BOT 73.00.B0015
          Firmware: MPK 08.00.B0015
             Other: 
             Other: 
     Supports 36 HID++ 2.0 features:
         0: ROOT                   {0000}   
         1: FEATURE SET            {0001}   
         2: DEVICE FW VERSION      {0003}   
         3: DEVICE NAME            {0005}   
         4: WIRELESS DEVICE STATUS {1D4B}   
         5: RESET                  {0020}   
         6: unknown:0007           {0007}   
         7: BATTERY STATUS         {1000}   
         8: REPROG CONTROLS V4     {1B04}   
         9: unknown:1C00           {1C00}   
        10: SWAP BUTTON            {2005}   
        11: CHANGE HOST            {1814}   
        12: unknown:1815           {1815}   
        13: NEW FN INVERSION       {40A2}   
        14: ENCRYPTION             {4100}   
        15: LOCK KEY STATE         {4220}   
        16: KEYBOARD DISABLE       {4521}   
        17: unknown:4531           {4531}   
        18: TOUCHPAD RAW XY        {6100}   
        19: GESTURE 2              {6501}   
        20: unknown:00C2           {00C2}   
        21: unknown:1802           {1802}   internal, hidden
        22: unknown:1803           {1803}   internal, hidden
        23: unknown:1806           {1806}   internal, hidden
        24: unknown:1813           {1813}   internal, hidden
        25: unknown:1805           {1805}   internal, hidden
        26: unknown:1830           {1830}   internal, hidden
        27: unknown:1890           {1890}   internal, hidden
        28: unknown:1891           {1891}   internal, hidden
        29: unknown:18A1           {18A1}   internal, hidden
        30: unknown:1DF3           {1DF3}   internal, hidden
        31: unknown:1E00           {1E00}   hidden
        32: unknown:1EB0           {1EB0}   internal, hidden
        33: unknown:1861           {1861}   internal, hidden
        34: unknown:18B0           {18B0}   internal, hidden
        35: unknown:1F11           {1F11}   internal, hidden
     Has 19 reprogrammable keys:
         0: unknown:00D1              , default: unknown:00AE                => unknown:00D1              
             is FN, FN sensitive, divertable, persistently divertable, pos:1, group:0, gmask:0
         1: unknown:00D2              , default: unknown:00AF                => unknown:00D2              
             is FN, FN sensitive, divertable, persistently divertable, pos:2, group:0, gmask:0
         2: unknown:00D3              , default: unknown:00B0                => unknown:00D3              
             is FN, FN sensitive, divertable, persistently divertable, pos:3, group:0, gmask:0
         3: unknown:00D6              , default: unknown:00B3                => unknown:00D6              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:4, group:0, gmask:0
         4: unknown:00C7              , default: unknown:00A3                => unknown:00C7              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:5, group:0, gmask:0
         5: unknown:00C8              , default: unknown:00A4                => unknown:00C8              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:6, group:0, gmask:0
         6: unknown:00E4              , default: Previous                    => unknown:00E4              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:7, group:0, gmask:0
         7: unknown:00E5              , default: Play/Pause                  => unknown:00E5              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:8, group:0, gmask:0
         8: unknown:00E6              , default: Next                        => unknown:00E6              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:9, group:0, gmask:0
         9: unknown:00E7              , default: Mute                        => unknown:00E7              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:10, group:0, gmask:0
        10: unknown:00E8              , default: Volume Down                 => unknown:00E8              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:11, group:0, gmask:0
        11: unknown:00E9              , default: Volume Up                   => unknown:00E9              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:12, group:0, gmask:0
        12: unknown:00BE              , default: unknown:009A                => unknown:00BE              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        13: unknown:00D5              , default: unknown:00B2                => unknown:00D5              
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        14: unknown:00BD              , default: unknown:0099                => unknown:00BD              
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        15: unknown:00BA              , default: unknown:0096                => unknown:00BA              
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        16: unknown:00D4              , default: unknown:00B1                => unknown:00D4              
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        17: unknown:00B7              , default: ShowUI                      => unknown:00B7              
             divertable, pos:0, group:0, gmask:0
        18: unknown:0034              , default: Do Nothing One              => unknown:0034              
             nonstandard, pos:0, group:0, gmask:0
     Battery: 90%, discharging.

```
Here is what the GUI shows me:
image

One little thing i'm seeing is that swap Fn keys when set on status "ON" deactivated the functions and functions normally. 
This means that when it is on, the system processes media functions without pressing the fn key.
I don't know if this is intended or a bug.

### LINKED ISSUE #755: M350 Pebble Mouse battery level reported as empty
**Information**

- Solaar version 1.0.1:
- Distribution: Fedora 31
- Kernel version: `Linux 5.6.6-200.fc31.x86_64 x86_64 GNU/Linux`
```
  2: Wireless Mouse M350
     Codename     : M350
     Kind         : mouse
     Wireless PID : 4080
     Protocol     : HID++ 4.5
     Polling rate : 8 ms (125Hz)
     Serial number: 2B442F40
     The power switch is located on the base.
     Battery status unavailable.
```

**Describe the bug**

The mouse battery is reported as empty. (reporting as requested in the devices readme file).

### LINKED ISSUE #756: MX Keys Information
**Information**

- Output of `solaar show` for the target device (if applicable):
```
  1: MX Keys Wireless Keyboard
     Codename     : MX Keys
     Kind         : keyboard
     Wireless PID : 408A
     Protocol     : HID++ 4.5
     Polling rate : 20 ms (50Hz)
     Serial number: F359F7AC
        Bootloader: BL1 08.00.B0011
          Firmware: MPK 12.00.B0011
             Other: 
     The power switch is located on the edge of top right corner.
     Supports 33 HID++ 2.0 features:
         0: ROOT                   {0000}   
         1: FEATURE SET            {0001}   
         2: DEVICE FW VERSION      {0003}   
         3: DEVICE NAME            {0005}   
         4: WIRELESS DEVICE STATUS {1D4B}   
         5: RESET                  {0020}   
         6: DEVICE FRIENDLY NAME   {0007}   
         7: BATTERY STATUS         {1000}   
         8: REPROG CONTROLS V4     {1B04}   
         9: CHANGE HOST            {1814}   
        10: HOSTS INFO             {1815}   
        11: BACKLIGHT2             {1982}   
        12: K375S FN INVERSION     {40A3}   
        13: ENCRYPTION             {4100}   
        14: LOCK KEY STATE         {4220}   
        15: KEYBOARD DISABLE       {4521}   
        16: MULTIPLATFORM          {4531}   
        17: DFUCONTROL SIGNED      {00C2}   
        18: DEVICE RESET           {1802}   internal, hidden
        19: unknown:1803           {1803}   internal, hidden
        20: CONFIG DEVICE PROPS    {1806}   internal, hidden
        21: unknown:1813           {1813}   internal, hidden
        22: OOBSTATE               {1805}   internal, hidden
        23: unknown:1830           {1830}   internal, hidden
        24: unknown:1890           {1890}   internal, hidden
        25: unknown:1891           {1891}   internal, hidden
        26: unknown:18A1           {18A1}   internal, hidden
        27: unknown:1DF3           {1DF3}   internal, hidden
        28: unknown:1E00           {1E00}   hidden
        29: unknown:1EB0           {1EB0}   internal, hidden
        30: unknown:1861           {1861}   internal, hidden
        31: unknown:1A20           {1A20}   internal, hidden
        32: unknown:18B0           {18B0}   internal, hidden
     Has 24 reprogrammable keys:
         0: unknown:00D1              , default: unknown:00AE                => unknown:00D1              
             nonstandard, divertable, persistently divertable, pos:0, group:0, gmask:0
         1: unknown:00D2              , default: unknown:00AF                => unknown:00D2              
             nonstandard, divertable, persistently divertable, pos:0, group:0, gmask:0
         2: unknown:00D3              , default: unknown:00B0                => unknown:00D3              
             nonstandard, divertable, persistently divertable, pos:0, group:0, gmask:0
         3: unknown:00C7              , default: unknown:00A3                => unknown:00C7              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:1, group:0, gmask:0
         4: unknown:00C8              , default: unknown:00A4                => unknown:00C8              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:2, group:0, gmask:0
         5: unknown:00E0              , default: unknown:00BF                => unknown:00E0              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:3, group:0, gmask:0
         6: unknown:00E1              , default: unknown:00C0                => unknown:00E1              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:4, group:0, gmask:0
         7: SHOW DESKTOP              , default: ShowDesktop                 => SHOW DESKTOP              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:5, group:0, gmask:0
         8: unknown:00E2              , default: unknown:00C1                => unknown:00E2              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:6, group:0, gmask:0
         9: unknown:00E3              , default: unknown:00C2                => unknown:00E3              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:7, group:0, gmask:0
        10: unknown:00E4              , default: Previous                    => unknown:00E4              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:8, group:0, gmask:0
        11: unknown:00E5              , default: Play/Pause                  => unknown:00E5              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:9, group:0, gmask:0
        12: unknown:00E6              , default: Next                        => unknown:00E6              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:10, group:0, gmask:0
        13: unknown:00E7              , default: Mute                        => unknown:00E7              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:11, group:0, gmask:0
        14: unknown:00E8              , default: Volume Down                 => unknown:00E8              
             is FN, FN sensitive, reprogrammable, divertable, persistently divertable, pos:12, group:0, gmask:0
        15: unknown:00E9              , default: Volume Up                   => unknown:00E9              
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        16: Calculator                , default: Calculator                  => Calculator                
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        17: unknown:00BF              , default: unknown:009B                => unknown:00BF              
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        18: unknown:00EA              , default: unknown:00C3                => unknown:00EA              
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        19: Lock PC                   , default: WindowsLock                 => Lock PC                   
             nonstandard, reprogrammable, divertable, persistently divertable, pos:0, group:0, gmask:0
        20: unknown:00EC              , default: unknown:00B8                => unknown:00EC              
             nonstandard, divertable, persistently divertable, pos:0, group:0, gmask:0
        21: unknown:00EB              , default: unknown:00B6                => unknown:00EB              
             nonstandard, divertable, persistently divertable, pos:0, group:0, gmask:0
        22: unknown:00DE              , default: Do Nothing One              => unknown:00DE              
             is FN, pos:0, group:0, gmask:0
        23: unknown:0034              , default: Do Nothing One              => unknown:0034              
             nonstandard, pos:0, group:0, gmask:0
     Battery: 20%, discharging.
```
Just leaving this here in case it's useful. Current version provides good support IMHO. Thank you.

### PR DESCRIPTION
**Pull request by pfps**
Also change wording at beginning a  bit.

Fixes #753 
Fixes #755 
Fixes #756

### CODE REVIEW

File: docs/devices.md [hunk 1]
```diff
@@ -58,11 +68,13 @@ You are able to read this feature using command-line interface of Solaar, but it
 | K400 Touch       | 4024 | 2.0   | yes     | FN swap                                 |
 | K400 Plus        | 404D | 2.0   |         | FN swap                                 |
 | K520             | 2011 | 1.0   | yes     | FN swap                                 |
+| K600 TV	   | 4078 | 2.0	  | yes	    | FN swap				      |
```
Comment (FFY00) [inline docs/devices.md:71]: You used tabs here.

File: docs/devices.md [hunk 2]
```diff
@@ -58,11 +68,13 @@ You are able to read this feature using command-line interface of Solaar, but it
...
 | K750 Solar       | 4002 | 2.0   | yes     | FN swap, Lux reading, light button      |
 | K780             | 405B | 2.0   | yes     | FN swap                                 |
 | K800 Illuminated | 2010 | 1.0   | yes     | FN swap, reprog keys, LEDs              |
 | K800 (new ver)   | 406E | 2.0   | yes     | FN swap                                 |
 | K830 Illuminated | 4032 | 2.0   | yes     | FN swap                                 |
+| MX Keys	   | 408A | 2.0	  | yes	    | 	 				      |
```
Comment (FFY00) [inline docs/devices.md:77]: Same.

File: docs/devices.md (hunk 1 above)
Comment (pfps) [reply docs/devices.md]: Done.  I keep forgetting this.

File: docs/devices.md (hunk 2 above)
Comment (pfps) [reply docs/devices.md]: Done

**Comment by pfps** [comment]
I also turned this very simple change into one commit

**Pull request merged**